ui/src/main/java/org/jtaccuino/core/ui/renderer/ListRenderer.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2025 JTaccuino Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jtaccuino.core.ui.renderer;
+
+import java.util.List;
+import java.util.Optional;
+import javafx.collections.FXCollections;
+import javafx.scene.Node;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TableView;
+import org.jtaccuino.core.ui.extensions.NodeRenderer;
+import org.jtaccuino.core.ui.extensions.NodeRenderer.Descriptor;
+
+@Descriptor(type = List.class, supportsArray = false)
+public class ListRenderer implements NodeRenderer {
+
+    @Override
+    public Optional<Node> render(Object object) {
+        var l = (List<?>) object;
+        if (!l.isEmpty() && l.getFirst().getClass().isRecord()) {
+            var tv = new TableView<Object>();
+            tv.getColumns().addAll(RenderHelper.tableColumnsFromRecordComponents(l.getFirst().getClass()));
+            tv.getItems().addAll(l);
+            tv.getColumns().forEach(c -> c.setPrefWidth(RenderHelper.COLUMN_PREF_WIDTH));
+            tv.setMaxHeight(RenderHelper.CELL_MAX_HEIGHT);
+            return Optional.of(tv);
+        } else {
+            var observableList = FXCollections.observableList(l.stream().map(String::valueOf).toList());
+            var lv = new ListView<String>(observableList);
+            lv.setMaxHeight(RenderHelper.CELL_MAX_HEIGHT);
+            return Optional.of(lv);
```

Comment:
Isn't this the same as 
```suggestion
        return Optional.of(RenderHelper.convertArrayToListView(l.tooArray()));
```